examples/proxy/record-validation/cluster-ip/README.md

```diff
@@ -0,0 +1,69 @@
+# Streams for Apache Kafka Proxy Record Validation, exposed using Cluster-IP
+
+In this example, an instance of Apache Kafka is deployed to an OpenShift cluster using Streams for Apache Kafka, alongside Apicurio Registry, which is also deployed to the cluster.
+
+The Streams for Apache Kafka Proxy is deployed with configuration to perform record validation.  The configuration ensures that 
+any records sent to a topic called `people` adhere to a `person` schema.
+
+Finally, Kafka command line tools are used to send valid and invalid records to the `people` topic, so the effects of the validation can be observed.
+so that the effects of the validation can be observed.
+
+# Prerequisites
+
+* Administrative access to the OpenShift Cluster being used to evaluate Streams for Apache Kafka Proxy
+* Streams for Apache Kafka Operator (installed cluster wide)
+* Red Hat build of Apicurio Registry Operator (installed cluster wide)
+* OpenShift CLI (`oc`)
+* `cURL`
+* Apache Kafka CLI tools (`kafka-topics.sh`, `kafka-console-producer.sh`, and `kafka-console-consumer.sh`) found in the `bin` directory of the Streams for Apache Kafka on RHEL distribution
+
+# Deploying the Example
+
+1. Deploy the Example:
+   ```sh
+   oc apply -k cluster-ip
```

Comment:
applying `cluster-ip` feels a bit odd in this example. To be clear I understand why: we need to expose the the proxy somehow! Would just be nice to hide that as some kind alias generic term...

Reply:
Raised as #513